aws-datastore/build.gradle

```diff
@@ -35,7 +36,10 @@ dependencies {
     testImplementation dependency.mockito
     testImplementation dependency.robolectric
     testImplementation dependency.androidx.test.core
+    testImplementation 'org.mockito:mockito-core:3.8.0'
```

Comment:
Can you define these in the root `build.gradle`, and then reference the variables from here?  Similar to how all the other dependencies are defined.